### PR TITLE
[Runtime env][Java] Add unit tests for specifying jars for tasks.

### PR DESCRIPTION
**Pull request by jovany-wang**
## Why are these changes needed?

It seems that we have already supported specifying java jars for normal tasks, this PR only needs to add unit tests for that.
## Related issue number
#21463

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: java/test/src/main/java/io/ray/test/RuntimeEnvTest.java [hunk 1]
```diff
@@ -193,4 +193,38 @@ public void testZipPackageInActor() {
...
+  private static boolean findClass(String className) {
+    try {
+      Class.forName(className);
+    } catch (ClassNotFoundException e) {
+      return false;
+    }
+    return true;
+  }
+
+  private static void testDownloadAndLoadPackageForTask(String url) {
```
Comment (SongGuyang) [inline java/test/src/main/java/io/ray/test/RuntimeEnvTest.java:206]: Can you add a test in which you input two urls and make sure both of them work?

Comment (jovany-wang) [reply java/test/src/main/java/io/ray/test/RuntimeEnvTest.java]: DONE

**Review by SongGuyang — APPROVED** [review]
LGTM.

**Comment by jovany-wang** [comment]
This PR only changes Java part and the CI failures happen on master.

**Comment by jovany-wang** [comment]
This PR only changes Java part and the CI failures happen on master.

**Pull request merged**